hack/build-rpms.sh

```diff
@@ -31,7 +31,7 @@ tar czf "${rpm_tmp_dir}/SOURCES/${OS_RPM_NAME}-test.tar.gz" \
 	.
 cp -r "${ci_data}/." "${rpm_tmp_dir}/SOURCES"
 
-echo "${OS_GIT_COMMIT}"
+sed -i 's/vault.centos.org\/centos\/$releasever/vault.centos.org\/centos\/7.7.1908/g' /etc/yum.repos.d/CentOS-Sources.r
```

Comment:
@umohnani8 err, doesn't look like the file name extension got mentioned correctly.

Reply:
should be `.repo`

Reply:
yup, typo - fixed!